### PR TITLE
Add documentation for adaptive phase filter parameter

### PR DESCRIPTION
**Pull request by hjkristenson**

### CODE REVIEW

File: docs/guides/insar_product_guide.md
Comment (hjkristenson) [inline docs/guides/insar_product_guide.md:1]: Are there any changes that should be made to lines 295-296?

Comment (jtherrmann) [reply docs/guides/insar_product_guide.md:1]: I think they're correct as-is, though we could expand the description of the "Phase filter parameter", although maybe "Dampening factor" is more general and applies to other types of phase filters? Though, at the moment, I don't believe we support any types of filters other than `adf`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.6.2]
+
+### Added
+* Added documentation to [InSAR Product Guide](docs/guides/insar_product_guide.md#adaptive-phase-filter) for the adaptive phase filter parameter option
+
```
Comment (jtherrmann) [inline CHANGELOG.md:14]: Do we also want to call out the extra re-structuring that you did?

File: docs/guides/insar_product_guide.md [hunk 2]
```diff
@@ -20,33 +20,58 @@ Users are cautioned to read the sections on [limitations](#limitations "Jump to
...
 
 {% block processing_options %}
-### Processing Options
+### Processing Options and Optional Files
 
-!!! tip "New Water Masking Approach"
+There are several options users can set when ordering InSAR On Demand products, including setting some processing parameters and selecting additional files to include in the output product package.
 
-    InSAR products can be phase unwrapped using a water mask. The option to "Apply water mask" sets pixels over coastal waters and large inland waterbodies as invalid for phase unwrapping. This reduces phase unwrapping errors and outputs a less noisy unwrapped interferogram.
+!!! tip "New: Adaptive Phase Filter parameter settings are now customizable!"
```
Comment (jtherrmann) [inline docs/guides/insar_product_guide.md:27]:
```suggestion
!!! tip "New: Adaptive Phase Filter parameter is now customizable!"
```

**Review by jtherrmann — COMMENTED** [review]
I rendered the docs locally and compared them to our production website, and confirmed that all of the re-structured sections render as I would expect.

File: docs/guides/insar_product_guide.md
Comment (hjkristenson) [reply docs/guides/insar_product_guide.md:1]: I assumed that "Dampening factor" came from the GAMMA documentation. I'm fine with this language as-is, but am happy to make a change if there's the potential for clearer or more understandable language.

File: CHANGELOG.md (hunk 1 above)
Comment (hjkristenson) [reply CHANGELOG.md:14]: I suppose that's fair.

Comment (hjkristenson) [reply CHANGELOG.md]: Addressed with [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/379/commits/0b137e5e37ddbd2129a104f59bfee1261c2c1167)

**Comment by hjkristenson** [comment]
Verified that trufflehog results are all storymap links

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**